### DISCUSSION TITLE
Is importing Jellyfin users the same as letting them login without importing?

### DISCUSSION THREAD

**Discussion by devanteweary** [seerr-team/seerr#619] [Q&A]
I have it set up to where users can use their JF account to log in to Jellyseerr. 
I noticed once they do log in the first time, it creates an account in Jellyseerr.

Is this the same as clicking the import JF users option?

Just wondering because if so, I'd like to do it to get a jump on adding their email address to their Jellyseerr account for notifications.

Thank you!

**Comment by fallenbagel** [comment]
Yes it's the same

**Reply by devanteweary** [reply]
OK cool. Thanks!

Just to let you know, I imported a user who had not yet logged into Jellyseerr and under Password, it has a warning that they don't have a password set yet.

Not sure if that'll affect anything!
They haven't tried to log in yet.

![image](https://github.com/Fallenbagel/jellyseerr/assets/29805497/487ab57d-08c7-497e-9ed8-a708803e4cee)

**Reply by fallenbagel** [reply]
> OK cool. Thanks!
> 
> Just to let you know, I imported a user who had not yet logged into Jellyseerr and under Password, it has a warning that they don't have a password set yet.
> 
> Not sure if that'll affect anything!
> They haven't tried to log in yet.
> 
> ![image](https://github.com/Fallenbagel/jellyseerr/assets/29805497/487ab57d-08c7-497e-9ed8-a708803e4cee)
> 

That is for local logins. If you're using jellyfin logins you can ignore that.
When a user login normally without being imported, their jellyfin password is added as their local password too but when you import there's no password to intercept and add so if you in the future intend to use local logins, then you'll have to ask them to add a local passwords.

But from your last response, I'm pretty sure you're using jellyfin logins anyways so you can safely ignore this. This is not an error. It's intended.
